L/libcvrpsep/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcvrpsep"
+version = v"0.0.1"
```

Comment:
The version of the package seems to be [v0.1.0](https://github.com/chkwon/CVRPSEP/blob/c4d1720ffafcc174e39f65c7a90146cb82b68e44/configure.ac#L5)
```suggestion
version = v"0.1.0"
```